src/compass/utils/h5_helpers.py

```diff
@@ -609,6 +618,18 @@ def corrections_to_h5group(parent_group, burst, cfg, rg_lut, az_lut,
                  f'Solid Earth tides (range) {desc}',
                  {'units': 'meters'}),
         ]
+        if weather_model_path is not None:
+            if 'wet' in delay_type:
+                correction_items.append(Meta('wet_los_troposphere_delay',
+                                             ds.GetRasterBand(5).ReadAsArray(),
+                                             f'Wet LOS troposphere delay {desc}',
+                                             {'units': 'meters'}))
+            if 'dry' in delay_type:
+                correction_items.append(Meta('dry_los_troposphere_delay',
```

Comment:
Does not need to be addressed in this PR, but reminding that we will need to rearrange the order of the corrections to write into the output HDF5 file because we have SET azimuth, and ionosphere not in place yet.
I'll submit a WIP PR to azimuth SET conversion, and let's utilize that PR to finalize all the corrections we have implemented so far.

Reply:
May be we can open an issue to track this down @seongsujeong ?

Reply:
The tasks for the finalization will be tracked here: 
#106